_build in alt-ergo-lib-2.3.2 release

I am trying to build alt-ergo-lib.2.3.2 with dune.2.6.0 and Multicore OCaml 4.10.0 as part of https://github.com/ocaml-bench/sandmark. We use bwrap sandboxing to build packages in a different directory, but, I see that the 2.3.2 release already has a _build/ directory in the release package. Hence, I get the following error when trying to compile alt-ergo-lib.2.3.2:

```
# [ERROR] ../sandmark/_opam exists, but does not appear to be a valid opam root. Please remove it and use `opam init', or specify a different `--root' argument
# Exception: End_of_file.
```

Is it possible to provide a tarball without the _build directory, or is there a way to clean the _build directory before running `configure`?

--- a/sources/CHANGES
+++ b/sources/CHANGES
@@ -1,6 +1,14 @@
 dev version
 ===========
 
+version 2.3.3, August 5, 2020
+================================
+
+* Minor release:
+
+  - Fix configure use of bash
+  - Add conflict instructions with the Alt-ergo-free package
+
 version 2.3.2, March 23, 2020
 ================================
 

--- a/sources/CHANGES
+++ b/sources/CHANGES
@@ -1,7 +1,7 @@
 dev version
 ===========
 
-version 2.3.3, August 5, 2020
+version 2.3.3, August 19, 2020
 ================================
 
 * Minor release:

--- a/sources/Makefile
+++ b/sources/Makefile
@@ -159,19 +159,22 @@ non-regression: all
 install: all
 	$(DUNE) install $(DUNE_FLAGS)		\
 		--prefix $(prefix)						\
-		--libdir $(libdir)
+		--libdir $(libdir)						\
+		--mandir $(mandir)
 
 # Use dune to uninstall the lib, bin, and gui
 uninstall:
 	$(DUNE) uninstall $(DUNE_FLAGS)	\
 		--prefix $(prefix)						\
-		--libdir $(libdir)
+		--libdir $(libdir)						\
+		--mandir $(mandir)
 
 # Install only the lib
 install-lib:
 	$(DUNE) install $(DUNE_FLAGS)		\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p alt-ergo-lib alt-ergo-lib
 
 # Uninstall the lib
@@ -179,6 +182,7 @@ uninstall-lib:
 	$(DUNE) uninstall $(DUNE_FLAGS)	\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p alt-ergo-lib alt-ergo-lib
 
 
@@ -187,6 +191,7 @@ install-parsers:
 	$(DUNE) install $(DUNE_FLAGS)		\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p alt-ergo-parsers alt-ergo-parsers
 
 # Uninstall the parsers
@@ -194,6 +199,7 @@ uninstall-parsers:
 	$(DUNE) uninstall $(DUNE_FLAGS)	\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p alt-ergo-parsers alt-ergo-parsers
 
 # Install only the binary
@@ -201,6 +207,7 @@ install-bin:
 	$(DUNE) install $(DUNE_FLAGS)		\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p alt-ergo alt-ergo
 
 # Uninstall the binary
@@ -208,6 +215,7 @@ uninstall-bin:
 	$(DUNE) uninstall $(DUNE_FLAGS)	\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p alt-ergo alt-ergo
 
 # Install only the binary
@@ -215,6 +223,7 @@ install-gui:
 	$(DUNE) install $(DUNE_FLAGS)		\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p altgr-ergo altgr-ergo
 
 # Uninstall the lib
@@ -222,6 +231,7 @@ uninstall-gui:
 	$(DUNE) uninstall $(DUNE_FLAGS)	\
 		--prefix $(prefix)						\
 		--libdir $(libdir)						\
+		--mandir $(mandir)						\
 		-p altgr-ergo altgr-ergo
 
 .PHONY: install uninstall

--- a/sources/alt-ergo-lib.opam
+++ b/sources/alt-ergo-lib.opam
@@ -20,6 +20,7 @@ depends: [
 ]
 conflicts: [
   "alt-ergo-free"
+  "alt-ergo-lib-free"
 ]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"

--- a/sources/alt-ergo-lib.opam
+++ b/sources/alt-ergo-lib.opam
@@ -18,6 +18,9 @@ depends: [
   "seq"
   "stdlib-shims"
 ]
+conflicts: [
+  "alt-ergo-free"
+]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/alt-ergo-lib.opam
+++ b/sources/alt-ergo-lib.opam
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "alt-ergo-lib"
-version: "2.3.2"
+version: "2.3.3"
 authors: "Alt-Ergo developers"
 maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
 license: "OCamlPro Non-Commercial Purpose License, version 1"

--- a/sources/alt-ergo-parsers.opam
+++ b/sources/alt-ergo-parsers.opam
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "alt-ergo-parsers"
-version: "2.3.2"
+version: "2.3.3"
 authors: "Alt-Ergo developers"
 maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
 license: "OCamlPro Non-Commercial Purpose License, version 1"

--- a/sources/alt-ergo-parsers.opam
+++ b/sources/alt-ergo-parsers.opam
@@ -18,6 +18,9 @@ depends: [
   "menhir"
   "stdlib-shims"
 ]
+conflicts: [
+  "alt-ergo-free"
+]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/alt-ergo-parsers.opam
+++ b/sources/alt-ergo-parsers.opam
@@ -18,9 +18,6 @@ depends: [
   "menhir"
   "stdlib-shims"
 ]
-conflicts: [
-  "alt-ergo-free"
-]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/alt-ergo.opam
+++ b/sources/alt-ergo.opam
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "alt-ergo"
-version: "2.3.2"
+version: "2.3.3"
 authors: "Alt-Ergo developers"
 maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
 license: "OCamlPro Non-Commercial Purpose License, version 1"

--- a/sources/alt-ergo.opam
+++ b/sources/alt-ergo.opam
@@ -15,6 +15,9 @@ depends: [
   "alt-ergo-lib" { = version }
   "alt-ergo-parsers" { = version }
 ]
+conflicts: [
+  "alt-ergo-free"
+]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/alt-ergo.opam
+++ b/sources/alt-ergo.opam
@@ -15,9 +15,6 @@ depends: [
   "alt-ergo-lib" { = version }
   "alt-ergo-parsers" { = version }
 ]
-conflicts: [
-  "alt-ergo-free"
-]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/altgr-ergo.opam
+++ b/sources/altgr-ergo.opam
@@ -18,9 +18,6 @@ depends: [
   "alt-ergo-lib" { = version }
   "alt-ergo-parsers" { = version }
 ]
-conflicts: [
-  "alt-ergo-free"
-]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/altgr-ergo.opam
+++ b/sources/altgr-ergo.opam
@@ -18,6 +18,9 @@ depends: [
   "alt-ergo-lib" { = version }
   "alt-ergo-parsers" { = version }
 ]
+conflicts: [
+  "alt-ergo-free"
+]
 homepage: "http://alt-ergo.ocamlpro.com/"
 dev-repo: "git+https://github.com/OCamlPro/alt-ergo.git"
 bug-reports: "https://github.com/OCamlPro/alt-ergo/issues"

--- a/sources/altgr-ergo.opam
+++ b/sources/altgr-ergo.opam
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "altgr-ergo"
-version: "2.3.2"
+version: "2.3.3"
 authors: "Alt-Ergo developers"
 maintainer: "OCamlPro <alt-ergo@ocamlpro.com>"
 license: "OCamlPro Non-Commercial Purpose License, version 1"

--- a/sources/configure
+++ b/sources/configure
@@ -1,3 +1,3 @@
-#!/bin/bash -e
+#!/bin/sh -ue
 
 ocaml unix.cma configure.ml "$@"

--- a/sources/configure.ml
+++ b/sources/configure.ml
@@ -110,6 +110,7 @@ let () =
   let () = Format.fprintf fmt "prefix=%s@." !prefix in
   let () = Format.fprintf fmt "libdir=%s@." !libdir in
   let () = Format.fprintf fmt "sharedir=%s@." !sharedir in
+  let () = Format.fprintf fmt "mandir=%s@." !mandir in
   let () = close_out ch in
   let () = Format.printf "done.@." in
   ()

--- a/sources/lib/util/version.ml
+++ b/sources/lib/util/version.ml
@@ -29,7 +29,7 @@
 (* WARNING: a "cut" is performed on the following file in the Makefile.
    DO NOT CHANGE its format *)
 
-let _version="2.3.2"
+let _version="2.3.3"
 
 let _release_commit = "(not released)"
 

